tools/depends/target/ffmpeg/Makefile

```diff
@@ -26,7 +26,10 @@ ifeq ($(CROSS_COMPILING), yes)
 endif
 ifeq ($(OS), linux)
   ffmpg_config += --target-os=$(OS) --cpu=$(CPU)
-  ffmpg_config += --enable-vdpau --enable-vaapi --enable-pic
+  ifneq ($(TARGET_PLATFORM),raspberry-pi)
+    ffmpg_config += --enable-vdpau
+  endif
+  ffmpg_config += --enable-vaapi --enable-pic
```

Comment:
rbpi gets build with depends and doesn't use https://github.com/xbmc/xbmc/blob/master/cmake/modules/FindFFMPEG.cmake#L244-L264.
If building depends fails you will notice it before building kodi.